### DISCUSSION TITLE
Frequency vs angular frequency (i18n / l10n)

### DISCUSSION THREAD

**Discussion by s-celles** [JuliaControl/ControlSystems.jl#917] [General]
Hello,

I'm not an english native speaker but noticed that frequency is used both to indicate frequency [Hz] and frequency [rad/s].
Don't you think that it could help to use the term [angular frequency](https://en.wikipedia.org/wiki/Angular_frequency) (especitally in `bodeplots`)
In France we have 2 words : fréquence and pulsation

Can we change

https://github.com/JuliaControl/ControlSystems.jl/blob/e8f68d6a621499f8ab751bf1198c8e3cd53df278/lib/ControlSystemsBase/src/plotting.jl#L300

and

https://github.com/JuliaControl/ControlSystems.jl/blob/e8f68d6a621499f8ab751bf1198c8e3cd53df278/lib/ControlSystemsBase/src/plotting.jl#L335

when using this function

Any opinion?

**Comment by baggepinnen** [comment]
Use of the term frequency in this context is rather well established, for example, both [matlab](https://se.mathworks.com/help/ident/ref/dynamicsystem.bode.html) and the [python control toolbox](https://python-control.readthedocs.io/en/0.9.4/generated/control.bode_plot.html) use the same terminology.

You can manually override the X-label when creating a plot by
```julia
bodeplot(P, xlabel="My choice of terminology")
```

**Comment by s-celles** [comment]
Great but I haven't seen how I can also change ylabel separately for Magnitude and Phase plot.

**Comment by baggepinnen** [comment]
```julia
julia> P = ssrand(1,1,4);

julia> bodeplot(P, ylabel = ["Mag" "Phase"])
```
![image](https://github.com/JuliaControl/ControlSystems.jl/assets/3797491/781df3ce-5e73-4c92-a1b0-73cf28840745)

**Reply by s-celles** [reply]
Thanks @baggepinnen
